core/types.mli

```diff
@@ -37,8 +38,11 @@ module Vars : sig
   type vars_list = (int * (flavour * kind * scope)) list
 end
 
+(* TODO return this to original (~S) *)
+type pp_policy = {quantifiers:bool; flavours:bool; hide_fresh:bool; kinds:string; effect_sugar:bool}
```

Comment:
It would be great if you created a module `Policy` for this datatype, e.g.
```ocaml
module Policy = struct
  type t = {quantifiers: bool; .... }
  let default_policy () = ...
end
```

Reply:
This is done now